configuration/pipeline_config.json

```diff
@@ -0,0 +1,77 @@
+
+    {"RapidProKey": "Rqa_S01E01 (Text) - unicef_covid19_som_s01e01_activation", "PipelineKey": "rqa_s01e01_raw", "IsActivationMessage": true},
+    {"RapidProKey": "Rqa_S01E01 (Run ID) - unicef_covid19_som_s01e01_activation", "PipelineKey": "rqa_s01e01_run_id"},
+    {"RapidProKey": "Rqa_S01E01 (Time) - unicef_covid19_som_s01e01_activation", "PipelineKey": "sent_on"},
+
+    {"RapidProKey": "Rqa_S01E02 (Text) - unicef_covid19_som_s01e02_activation", "PipelineKey": "rqa_s01e02_raw", "IsActivationMessage": true},
+    {"RapidProKey": "Rqa_S01E02 (Run ID) - unicef_covid19_som_s01e02_activation", "PipelineKey": "rqa_s01e02_run_id"},
+    {"RapidProKey": "Rqa_S01E02 (Time) - unicef_covid19_som_s01e02_activation", "PipelineKey": "sent_on"},
+
+    {"RapidProKey": "Rqa_S01E03 (Text) - unicef_covid19_som_s01e03_activation", "PipelineKey": "rqa_s01e03_raw", "IsActivationMessage": true},
+    {"RapidProKey": "Rqa_S01E03 (Run ID) - unicef_covid19_som_s01e03_activation", "PipelineKey": "rqa_s01e03_run_id"},
+    {"RapidProKey": "Rqa_S01E03 (Time) - unicef_covid19_som_s01e03_activation", "PipelineKey": "sent_on"},
+
+    {"RapidProKey": "Rqa_S01E04 (Text) - unicef_covid19_som_s01e04_activation", "PipelineKey": "rqa_s01e04_raw", "IsActivationMessage": true},
+    {"RapidProKey": "Rqa_S01E04 (Run ID) - unicef_covid19_som_s01e04_activation", "PipelineKey": "rqa_s01e04_run_id"},
+    {"RapidProKey": "Rqa_S01E04 (Time) - unicef_covid19_som_s01e04_activation", "PipelineKey": "sent_on"},
+
+    {"RapidProKey": "Gender (Text) - unicef_covid19_som_demogs", "PipelineKey": "gender_raw"},
+    {"RapidProKey": "Gender (Time) - unicef_covid19_som_demogs", "PipelineKey": "gender_time"},
+
+    {"RapidProKey": "Age (Text) - unicef_covid19_som_demogs", "PipelineKey": "age_raw"},
+    {"RapidProKey": "Age (Time) - unicef_covid19_som_demogs", "PipelineKey": "age_time"},
+
+    {"RapidProKey": "Recently_Displaced (Text) - unicef_covid19_som_demogs", "PipelineKey": "recently_displaced_raw"},
+    {"RapidProKey": "Recently_Displaced (Time) - unicef_covid19_som_demogs", "PipelineKey": "recently_displaced_time"},
+
+    {"RapidProKey": "Household_Language (Text) - unicef_covid19_som_demogs", "PipelineKey": "household_language_raw"},
+    {"RapidProKey": "Household_Language (Time) - unicef_covid19_som_demogs", "PipelineKey": "household_language_time"},
+
+    {"RapidProKey": "District (Text) - unicef_covid19_som_demogs", "PipelineKey": "location_raw"},
+    {"RapidProKey": "District (Time) - unicef_covid19_som_demogs", "PipelineKey": "location_time"}
+
+  ],
+  "ProjectStartDate": "2020-07-03T00:16:30+03:00",
+  "ProjectEndDate": "2100-01-01T00:00:00+03:00",
+  "FilterTestMessages": true,
+  "MoveWSMessages": true,
+  "DriveUpload": {
+    "DriveCredentialsFileURL": "gs://avf-credentials/pipeline-runner-service-acct-avf-data-core-64cc71459fe7.json",
+    "ProductionUploadPath": "analysis_outputs/unicef_covid19_som_production.csv",
```

Comment:
I think analysis_outputs needs to be prefixed with the project name, otherwise we'll end up with multiple directories called analysis_outputs and the drive upload scripts won't know which to update

Reply:
updated thanks.